packages/ws-server/tests/acceptance/estimateGas.spec.ts

```diff
@@ -51,6 +53,26 @@ describe('@release @web-socket eth_estimateGas', async function () {
 
   beforeEach(async () => {
     ethersWsProvider = new ethers.WebSocketProvider(WsTestConstant.WS_RELAY_URL);
+    const wallet = new ethers.Wallet(accounts[0].wallet.privateKey, ethersWsProvider);
+    const basicContractFactory = await new ethers.ContractFactory(
+      basicContractJson.abi,
+      basicContractJson.bytecode,
+      wallet,
+    );
+
+    let retries = 0;
+    while (retries < 3) {
```

Comment:
Q: Curious why we need a polling loop here? It shouldn't take that long to deploy a contract, should it?

Reply:
There are intermittent errors related to incomplete deployments if the retries are not there.

Reply:
I just tried running this test without the polling loop and recognize the error `"message": "Error invoking RPC: \"Requested resource not found. address...` If this is the same error you've been encoutering, I believe the problem is not in the deployment but in the acocunt creation using `servicesNode` at line 45. That's why I added sleepers like [this here](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/ws-server/tests/acceptance/call.spec.ts#L69) to wait for the accounts to be propagated.

Reply:
Okay thanks.  I updated it to poll for the account on the mirrornode.